packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -199,6 +199,42 @@ class GuildBanManager extends CachedManager {
     await this.client.rest.delete(Routes.guildBan(this.guild.id, id), { reason });
     return this.client.users.resolve(user);
   }
+
+  /**
+   * Bulk ban users from a guild, and optionally delete previous messages sent by the banned users.
+   * @param {Collection<Snowflake, User|UserResolvable>|UserResolvable[]} users The users to ban
```

Comment:
`User` is a `UserResolvable`
```suggestion
   * @param {Collection<Snowflake, UserResolvable>|UserResolvable[]} users The users to ban
```